config/environments/test.rb

```diff
@@ -39,4 +39,5 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.stripe.secret_key = ENV['STRIPE_SECRET_KEY']
```

Comment:
To be able test stripe, we need a secret api key to test it. In this file we set it for the test environment.